### ISSUE TITLE
Modules should be watched and compiled individually?

### ISSUE THREAD

**Issue by jbmoelker** [voorhoede/front-end-guide#15]

**Comment by jpsc** [comment]
I don't think that modules can live alone. It's pretty much impossible to output an module that includes for example all the necessary styles without actually adding them specifically for the individual output
